src/psyclone/psyGen.py

```diff
@@ -3345,12 +3346,13 @@ def has_inc_arg(self):
         return False
 
     def unique_modified_args(self, arg_type):
-        '''Return all unique arguments of type arg_type from Kernels in this
-        loop that are modified.
+        '''Return all unique arguments of the given type from kernels inside
+        this loop that are modified.
+
         :param str arg_type: the type of kernel argument (e.g. field, \
                              operator) to search for.
-        :returns: all unique arguments of type arg_type from Kernels in this
-        loop that are modified.
+        :returns: All unique arguments of the given type from kernels inside \
```

Comment:
Lowercase "all" please.

Reply:
Also, lowercase "list" below and no full stop at the end.

Reply:
fixed